app/controllers/application_controller.rb

```diff
@@ -1,17 +1,7 @@
 class ApplicationController < ActionController::Base
+
   include Lev::HandleWith
 
   respond_to :html
 
-  layout 'application'
-
-  # skip all filters defined so far
-  skip_filter *_process_action_callbacks.map(&:filter), only: [:routing_error]
```

Comment:
Moved to ExceptionsController.

Basically when we find an exception we don't care if the user is logged in, we just want to display the error page.